appserver/tests/appserv-tests/devtests/connector/v3/run_test.sh

```diff
@@ -51,9 +51,9 @@ test_run(){
   echo $ROOT
   cd $APS_HOME/../v2-tests/appserv-tests
   ant startDomain startDerby
-  antTarget="clean-all clean all report"
   cd $ROOT
-  time ant $antTarget | tee $TEST_RUN_LOG
+  echo Running target: $TARGET
+  time ant clean-all clean $TARGET report | tee $TEST_RUN_LOG
   antStatus=$?
   cp connector.output tests-run.log
   cd $APS_HOME/../v2-tests/appserv-tests
```

Comment:
/v2-tests/appserv-tests folder would be deleted soon. That is the whole purpose of moving test from v2-tests/appserv-tests to appserv-tests . Please remove this reference